docs/README.md

```diff
@@ -6,9 +6,9 @@ Bitski JS SDK is a JavaScript library that connects your DApp with a user, a wal
 
 ### Getting Started
 
-In order to use Bitski you will need an account. You can set one up here: [https://www.bitski.com/account](https://www.bitski.com/account).
+In order to use Bitski you will need an account. You can set one up here: [https://www.bitski.com/signup](https://www.bitski.com/signup).
```

Comment:
Did we want this to be develop.bitski.com/signup?

Reply:
Or is that a subsequent PR once we have that in place

Reply:
Yeah, we can link to that once it exists.

Reply:
👍